src/styles.ts

```diff
@@ -88,7 +88,7 @@ export const baseVariables = css`
     width: 100%;
     max-width: 1280px;
   }
-}`;
```

Comment:
This is a drive-by. Not necessary since the new code doesn't use `baseVariables` anymore, but I found it, so why not fix it